docs/src/components/ColorSwatch/ColorSwatch.js

```diff
@@ -48,13 +49,15 @@ ColorSwatch.propTypes = {
   name: PropTypes.string.isRequired,
   className: PropTypes.string,
   color: PropTypes.string,
+  textColor: PropTypes.string,
```

Comment:
It seems that `textColor` and `whiteColor` are both solving the same problem. Can we remove `whiteColor` from the swatch and replace all usages with `textColor={colorWhite}`?

Reply:
Sure why not 😄